deafrica/monitoring/latency_check.py

```diff
@@ -170,5 +172,7 @@ def cli(
     if version:
         click.echo(__version__)
     res = latency_checker(
-        satellite=satellite, latency=latency, notification_slack_url=slack_url
+        satellite=satellite,

```

Comment:
@Zintle95 you need to pass through the new parameters (`Bucket` and `Prefix`) to the `latency_checker` function and then also call your new function (`s3_latency`) within `latency_checker` with these new parameters and the `latency` parameter (right now hard-coded to 3 days). Also note that in `latency_checker` your `s3_latency` variable is currently being used but is unassigned. It must be assigned a value by calling the `s3_latency` function. To avoid confusion you can also consider giving the variable a different name than the function